theseus/geometry/se2.py

```diff
@@ -92,37 +94,25 @@ def update_from_rot_and_trans(self, rotation: SO2, translation: Point2):
     # From https://github.com/strasdat/Sophus/blob/master/sophus/se2.hpp#L160
```

Comment:
How does the approximation you used below relate to the one used in Sophus? Can you change the reference to point to the source for the expressions below?

Reply:
I forgot to remove this line. The implementation of `so2.exp_map` and `so2.log_map` is a rewrite. I will add some comments.

Reply:
Yes, the tactile pushing example successfully computes the gradient.